docs/source/workflow_outputs.rst

```diff
@@ -403,6 +403,17 @@ Annual hot water uses are listed below.
    Hot Water: Distribution Waste (gal) 
    ===================================  ====================
 
+Resilience Hours
+~~~~~~~~~~~~~~~~
+
+Resilience hours by system are listed below.
+
+   ===================================  ====================
+   Type                                 Notes
+   ===================================  ====================
+   Resilience: Battery (hr)             Use the battery state of charge and the load profile to figure out how long the battery would have been able to support all of the electrical loads. This treats all of the loads as "critical loads" that would be supported during an outage. Calculation is done for every timestep and then averaged, which assumes an outage is equally likely to occur every hour of the year. Resilience hours are set to 0 for any timestep where the battery is not charged, even if there is sufficient PV to power the building.
```

Comment:
Simplify the note to a shorter description and move the rest of it into a table footnote. I haven't looked but I assume the table will look ridiculous with this much text.